dev-docs/examples/multi-format-example.md

```diff
@@ -7,152 +7,85 @@ sidebarType: 1
 
 about:
 - A multi-format ad unit allows you to receive any combination of banner, video, or native demand
+- A multi-format ad unit has two methods - a simple setup with in-renderer integration that does not require complex configuration on the ad server side, and format switching on the ad server side
 - Any bidder that supports at least one of the listed media types can participate in the auction for that ad unit
 - For engineering setup instructions, see <a href="/dev-docs/show-multi-format-ads.html">Show Multi-Format Ads</a>
-- For ad ops setup instructions, see <a href="/adops/step-by-step.html">Google Ad Manager with Prebid Step by Step</a>
+- In this example, to keep it simple, ad rendering is performed using `pbjs.renderAd`
+- InRenderer.js, an open-source renderer optimized for in-renderer integration, is used as the renderer for in-renderer integration. For more information, see <a href="https://github.com/hogekai/in-renderer-js" target="_blank">InRenderer.js documentation</a>
 
 ---
 
-## Basic Prebid.js Example
+## Example
 
-{% capture htmlCodePrebid %}<h3>div-banner-native-1</h3>
-<div id='div-banner-native-1'>
-<p>No response</p>
-<script type='text/javascript'>
-    googletag.cmd.push(function() {
-        googletag.display('div-banner-native');
-    });
+{% capture htmlCodePrebid %}
+<script src="https://cdn.michao-ssp.com/original/prebid.js"></script>
```

Comment:
I would leave this PR open until https://github.com/prebid/Prebid.js/pull/12507 is merged and it can be replaced with "not-for-prod" distribution, so you don't have to maintain this distribution